db/migrate/20150930183738_migrate_content_hosts.rb

```diff
@@ -0,0 +1,193 @@
+class MigrateContentHosts < ActiveRecord::Migration
+  class Katello::HostCollectionAssociation < ActiveRecord::Base
+    self.table_name = "katello_system_host_collections"
+  end
+
+  class Katello::ErrataAssociation < ActiveRecord::Base
+    self.table_name = "katello_system_errata"
+  end
+
+  class Katello::RepositoryAssociation < ActiveRecord::Base
+    self.table_name = "katello_system_repositories"
+  end
+
+  class Katello::System < ActiveRecord::Base
+    def backend_data
+      @data ||= ::Katello::Resources::Candlepin::Consumer.get(uuid)
+    end
+
+    def facts
+      backend_data[:facts]
+    end
+
+    self.table_name = "katello_systems"
+
+    has_many :system_activation_keys, :class_name => "Katello::SystemActivationKey", :dependent => :destroy
+    has_many :activation_keys,
+             :through => :system_activation_keys,
+             :after_add    => :add_activation_key,
+             :after_remove => :remove_activation_key
+    belongs_to :content_view, :inverse_of => :systems
+    belongs_to :environment, :class_name => "Katello::KTEnvironment", :inverse_of => :systems
+    has_many :errata_associations, :class_name => "Katello::ErrataAssociation", :dependent => :destroy
+    has_many :repository_associations, :class_name => "Katello::RepositoryAssociation", :dependent => :destroy
+    has_many :host_collections, :class_name => "Katello::HostCollectionAssociation", :dependent => :destroy
+  end
+
+  def logger
+    Rails.logger
+  end
+
+  def create_content_aspect(host, system)
+    logger.info("Creating content aspect for host #{host.name}.")
+    host.content_aspect = ::Katello::Host::ContentAspect.new(:content_view => system.content_view, :lifecycle_environment => system.environment)
+  end
```

Comment:
as we chatted, content aspect needs 'bound_repositories' and 'appliacable_errata' also copied from the system to the content_aspect